code/processes/dqe.q

```diff
@@ -113,8 +113,24 @@ postback:{[runtype;idnum;proc;params;result]
     .dqe.updresultstab[runtype;idnum;.z.p;first result;result[1];`complete;params;proc]];
   }
 
+containerfn:{[funct;names;vars]
+  if[any b:where 11h=abs type each vars;
+    startvars:vars b;
+    names:names b;
+    finvars:startvars b2:where startvars in tables[];
+    names:names b2;
+    if[count finvars;
+      finvars:finvars b3:where `date in ' cols each finvars;
+      names:names b3;
+      names set ' value each ("select from " ,/: (string finvars)),\:" where date=.z.p-1"];
```

Comment:
```suggestion
      names set' {select from x where date=.z.p-1}'[finvars];
```

should work here I think?

also should that be `.z.d` perhaps? what about if this is run on Mondays, should it be looking back to Friday?

Reply:
I am going to replace `.z.d-1` with the most recent date in the date variable stored in the HDB.

Reply:
Just realised that I typed `.z.p` instead of `.z.d`. 
As said though it will be replaced soon enough :)